CRM/Admin/Form/MailSettings.php

```diff
@@ -109,11 +127,15 @@ public function getDefaultEntity() {
   public function setDefaultValues() {
     $defaults = parent::setDefaultValues();
 
-    // Set activity status to "Completed" by default.
-    if ($this->_action != CRM_Core_Action::DELETE &&
-      (!$this->_id || !CRM_Core_DAO::getFieldValue('CRM_Core_BAO_MailSettings', $this->_id, 'activity_status'))
-    ) {
+    if (!$this->_id) {
+      $defaults['is_ssl'] = TRUE;
```

Comment:
Added sensible defaults for new mail accounts here to check Use SSL and select email-to-activity instead of bounce processing (of which there can only be one anyways).